### DISCUSSION TITLE
🗓️ Uniting through Actions - Community Check-In

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#64193] [Actions]
Hello and welcome to another check-in. We’re back with the latest from the Actions world. 🌎 This week, I’ll share some of the new additions to Actions, thank yous to our incredible community members, and an opportunity to share your valuable feedback.

## ✨ Meet the Community Champions: Shoutout 📢  to these members

- Huge round of applause 👏 for [@nirav-gajera](https://github.com/nirav-gajera) for being a top contributor in the Actions space. Though they have contributed to multiple discussions, the one that really stands out is where they helped us understand [how to allow anyone in the repo to trigger an Actions workflow.](https://github.com/orgs/community/discussions/63336) 
- [@saragon](https://github.com/saragon) reported getting to an[ incorrect workflow path](https://github.com/orgs/community/discussions/62303) while triggering a workflow in solidity `.sol files`, our superstar ⭐ [@Abdullahxz](https://github.com/Abdullahxz) provided a perfect solution suggesting the use of  " **.sol " instead of calling the workflow. Kudos for providing the [Actions documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths) too :) 
- Thank You [@D3vil0p3r](https://github.com/D3vil0p3r) for coming back to their [discussion](https://github.com/orgs/community/discussions/61493) where they originally failed to create shim task but with a thorough research, got to the root cause. 👍 for sharing the solution with us - what an incredible self-solve! 💯
- Now, this one is special ✨ - [@ghoullier](https://github.com/ghoullier) wanted to automatically close waiting jobs related to deployment review when a pull request is closed/merged, but couldn’t get it to work. However, they solved it by writing a [new Action](https://github.com/orgs/community/discussions/62390#discussioncomment-6620966). Great job! 

Another big Thank You to this month’s highlighted members 🌟✨

| [@nirav-gajera](https://github.com/nirav-gajera) | [@saragon](https://github.com/saragon) | [@ Abdullahxz](https://github.com/Abdullahxz) | [@D3vil0p3r](https://github.com/D3vil0p3r) | [@ghoullier](https://github.com/ghoullier) |
|--|--|--|--|--|
| <img src="https://avatars.githubusercontent.com/u/127472114?v=3?s=100" width="100px;" alt="nirav-gajera" style="max-width: 100%;"> | <img src="https://avatars.githubusercontent.com/u/13812458?v=3?s=100" width="100px;" alt="saragon" style="max-width: 100%;"> | <img src="https://avatars.githubusercontent.com/u/55325434?v=3?s=100" width="100px;" alt="Abdullahxz" style="max-width: 100%;"> | <img src="https://avatars.githubusercontent.com/u/83867734?v=3?s=100" width="100px;" alt="D3vil0p3r" style="max-width: 100%;"> | <img src="https://avatars.githubusercontent.com/u/2315749?v=3?s=100" width="100px;" alt="ghoullier" style="max-width: 100%;"> |

## 🪐Universe 2023 - Are you ready for it?

Universe 2023 tickets are now available at a super-early bird discount! [Learn more about Universe](https://github.com/orgs/community/discussions/63834).

Build for today, transform for tomorrow at GitHub Universe 2023: Read our [blogpost](https://github.blog/2023-08-15-build-for-today-transform-for-tomorrow-at-github-universe-2023/) to get even more excited about Universe 2023.

### [Actions runs triggered from tags or forks with the same name as a protected branch will now be blocked](https://github.blog/changelog/2023-08-10-actions-runs-triggered-from-tags-or-forks-with-the-same-name-as-a-protected-branch-will-now-be-blocked/)

GitHub environments can be configured with deployment branch policies to allow-list the branches that can deploy to them. We are now security hardening these branch policies further by blocking runs triggered from forks with branches that match the protected branch name. 

Learn more about configuring environments with [deployment protection rules](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#deployment-branches) to set up rigorous and streamlined guardrails for your deployments.

### 👀 [Required Workflows will move to Repository Rules](https://github.blog/changelog/2023-08-02-github-actions-required-workflows-will-move-to-repository-rules/)

**We will be moving the private beta of required workflows on GitHub Actions to Repository Rules** to give organization administrators a powerful way to protect their repositories with added feature benefits including unified configuration, dry running workflow rules, branch targeting, and a consistent UI experience.

Existing Actions Required Workflows private beta users will continue to have access until October 18th, 2023, allowing them time to adapt to the forthcoming changes. To know more about how this change will impact you and what you need to do, check out the [changelog post](https://github.blog/changelog/2023-08-02-github-actions-required-workflows-will-move-to-repository-rules/). 

### [[Private Beta] Jenkins Scripted Pipeline Migrations](https://github.blog/changelog/2023-08-04-private-beta-jenkins-scripted-pipeline-migrations/) 

**Support for migrating Jenkins Scripted Pipelines to GitHub Actions is now available** as a private beta! 🎉  If you use Scripted Pipelines in your Jenkins instances, you can now automate the migration of your pipelines to GitHub Actions using [GitHub Actions Importer](https://docs.github.com/en/actions/migrating-to-github-actions/automated-migrations/automating-migration-with-github-actions-importer).

To get started, please reach out to your GitHub account manager or [contact our sales team](https://github.com/enterprise/contact)! 🚀

### 🆕  [New dashboard view for deployments across environments (public beta)](https://github.blog/changelog/2023-07-26-github-actions-new-dashboard-view-for-deployments-across-environments-public-beta/)

![Screenshot 2023-08-16 at 2 44 03 PM](https://github.com/community/community/assets/95834130/028bc21e-c759-4389-9161-8fd3a4735a93)

Developers and DevOps managers can now view and track the full history of deployments in a repository or filter them across environments. **Learn more about viewing deployments** in your repository [through our documentation ](http://docs.github.com/en/actions/deployment/managing-your-deployments/viewing-deployment-history)and [watching this video.](https://www.youtube.com/watch?v=ApMbLyPziZ8)

❓ Which one is your favorite new feature in Actions? Let us know in the comments below. :shipi

**Comment by rkostyantyn** [comment]
Hello,
I wonder if the jenkins-scripted-pipelines feature should already work (albeit experimentally) in the latest public release? Because I see no difference in output between files with or without this feature enabled.
Logs differ only with lines:
`jenkins-scripted-pipelines (enabled):
        Experimentally attempts to transform Groovy scripted pipelines. To join the private beta and provide feedback, you can sign up at https://gh.io/gai-chat-with-us!
` 
Thank you)
